language-service/language_service/dto/__init__.py

```diff
@@ -0,0 +1,33 @@
+"""
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First line should end with a period, question mark, or exclamation point (not 's')](https://app.codacy.com/manual/lucaskjaero/foreign-language-reader/pullRequest?prid=5112709)